gatsby-node.ts

```diff
@@ -202,7 +202,7 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
       });
   });
   const result = await graphql(`
-    query {
+    query stuff {
```

Comment:
wait why

Reply:
gatsby was complaining about "anonymous queries"

Reply:
odd...maybe put a more descriptive name then 😅

Reply:
will fix when i get the rest of the stuff working lmao